lib/AST/ModuleNameLookup.cpp

```diff
@@ -153,9 +153,8 @@ void ModuleNameLookup<LookupStrategy>::lookupInModule(
       resolutionKind == ResolutionKind::Overloadable) {
     // If we only found top-level functions, keep looking, since we may
     // find additional overloads.
-    if (std::find_if(decls.begin() + initialCount, decls.end(),
-                     [](ValueDecl *VD) { return !isa<FuncDecl>(VD); })
-        == decls.end())
+    if (std::none_of(decls.begin() + initialCount, decls.end(),
+                     [](ValueDecl *VD) { return isa<FuncDecl>(VD); })
```

Comment:
You inverted the logic here. "fail to find not-a-FuncDecl" is "none are not FuncDecls" or "all are FuncDecls".

Reply:
Yeah, oops. I think all_of() is best here.